packages/tsconfig/package.json

```diff
@@ -1,5 +1,10 @@
 {
   "name": "@turbo/tsconfig",
   "version": "0.0.0",
-  "private": true
+  "private": true,
+  "exports": {
+    ".": "./src/index.js",
+    "./base.json": "./base.json",
+    "./library.json": "./library.json"
+  }
```

Comment:
This shouldn't be required and won't do anything without swapping to ESM - does it help the ts server resolve the configs?

Reply:
ahh yes yes, good catch.  this was while I was debugging (what I didn't know at the time was) [the ts-node bug I mentioned](https://github.com/TypeStrong/ts-node/pull/2091).  basically.... well.. nevermind.  it was a whole thing to figure out what was happening and this now shouldn't be needed.

fixed in 7331be9824c05e8889d80baf451702bc872c3311

Reply:
Sounds good!